testcafe/tests/drive/classification_scenario.js

```diff
@@ -0,0 +1,395 @@
+  )
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_CREATE_FOLDER}-Folder1-2`
+  )
+
+  //Create 2nd Folder
+  await privateDrivePage.addNewFolder(
+    `${FEATURE_PREFIX}-Folder2`,
+    `${FEATURE_PREFIX}/${TEST_CREATE_FOLDER}-Folder2-1`
+  )
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_CREATE_FOLDER}-Folder2-2`
+  )
+  console.groupEnd()
+})
+
+test(`${TEST_UPLOAD}`, async t => {
+  console.group(
+    `↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_UPLOAD} (in "${FEATURE_PREFIX}-Folder2" folder)`
+  )
+  await privateDrivePage.goToFolder(`${FEATURE_PREFIX}-Folder2`)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_UPLOAD}-1`
+  )
+  await privateDrivePage.uploadFiles([`${FILE_FROM_ZIP_PATH}/${FILE_PDF}`])
+  await t.wait(5000)
+  await t.fixtureCtx.vr.setMaksCoordonnates(maskDriveFolderWithDate)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_UPLOAD}-2`,
+    true
+  )
+  console.groupEnd()
+})
+
+test(`${TEST_UPLOAD_DD}`, async t => {
+  console.group(
+    `↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_UPLOAD_DD} (in "${FEATURE_PREFIX}-Folder2" folder)`
+  )
+  await privateDrivePage.goToFolder(`${FEATURE_PREFIX}-Folder2`)
+  await privateDrivePage.uploadFiles([`${FILE_FROM_ZIP_PATH}/${FILE_TXT}`])
+  await t.wait(5000)
+  await t.fixtureCtx.vr.setMaksCoordonnates(maskDriveFolderWithDate)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_UPLOAD_DD}-1`,
+    true
+  )
+  console.groupEnd()
+})
+
+//************************
+//Tests when authentified
+//************************
+fixture`${FIXTURE_CLASSIFICATION}`.page`${TESTCAFE_DRIVE_URL}/`
+  .before(async ctx => {
+    await initVR(ctx, SLUG, FIXTURE_CLASSIFICATION)
+  })
+  .beforeEach(async t => {
+    console.group(`\n↳ ℹ️  Loggin & Initialization`)
+    await t.useRole(driveUser)
+    await privateDrivePage.waitForLoading()
+    console.groupEnd()
+  })
+  .after(async ctx => {
+    await ctx.vr.checkRunStatus()
+  })
+
+test(`${TEST_RENAME_FILE}`, async t => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_RENAME_FILE}`)
+  const [fileName, ext] = FILE_PDF.split('.')
+  //set mask only once, all screen in this test use the same mask
+  await t.fixtureCtx.vr.setMaksCoordonnates(maskDriveFolderWithDate)
+
+  await privateDrivePage.goToFolder(`${FEATURE_PREFIX}-Folder2`)
+  await privateDrivePage.renameElementByName(
+    FILE_PDF,
+    `${fileName}2.${ext}`,
+    `${FEATURE_PREFIX}/${TEST_RENAME_FILE}-1`,
+    { exitWithEnter: true }
+  )
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_RENAME_FILE}-1-rename2`,
+    true
+  )
+
+  await privateDrivePage.renameElementByName(
+    `${fileName}2.${ext}`,
+    FILE_PDF,
+    `${FEATURE_PREFIX}/${TEST_RENAME_FILE}-2`,
+    { exitWithEnter: false }
+  )
+
+  await t.fixtureCtx.vr.setMaksCoordonnates(maskDriveFolderWithDate)
```

Comment:
A few lines earlier a comment says we're only setting the mask once, and here we're setting it again?